factories/house.py

```diff
@@ -0,0 +1,19 @@
+import random
+
+import factory
+
+from house.models import House, Country, City, Job
+
+
+class HouseFactory(factory.Factory):
+    class Meta:
```

Comment:
How is this class expressed?

Reply:
We are using this in the create_mock_data script